kustomize/traffic/clusterScoped/kustomization.yaml

```diff
@@ -1,5 +1,25 @@
 bases:
-- ../base
+- ../namespaceScoped
 
```

Comment:
I think we need to add:

`namespace: default`

Reply:
This will install it everything in the `default` namespace only when `clusterScoped` is used.